### DISCUSSION TITLE
Visualizing programmatically created workflows inside Dashboard?

### DISCUSSION THREAD

**Discussion by StefanCatalinP** [elsa-workflows/elsa-core#3396] [Q&A] [answered]
Does the feature of visualizing in the Dashboard of programmatically created workflows currently exist in Elsa 2?
If not is it planned for a future release perhaps? 

This has been referenced previously in this discussion as something planned for Elsa 2, but I'm finding trouble finding any mention of it in the released documentation.
https://github.com/elsa-workflows/elsa-core/discussions/606

**Comment by FransVanEk** [comment, marked as answer]
Hi,

if you mean that the programmatically created workflows should appear in the registry of the designer then the answer would be yes. It is possible and working. 

If this doesn't answer your question, can you elaborate a bit further?

**Reply by StefanCatalinP** [reply]
Thank you. This works. I was expecting it to show under the definitions tab instead of the registry.
